firefox-ios/firefox-ios-tests/Tests/XCUITests/HomepageSearchBarTests.swift

```diff
@@ -0,0 +1,511 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+final class HomepageSearchBarTests: FeatureFlaggedTestBase {
+    func test_homepageSearchBar_experimentOff_doesNotShowSearchBar() {
```

Comment:
We name the tests with the `_experimentOff` and `_experimentOn` suffix.

Reply:
updated!